express-notes-api/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json')
+const notes = data.notes;
+const fs = require('fs');
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const arr = [];
+  for (const x in notes){
+    arr.push(notes[x]);
+  }
+  res.status(200).json(arr);
+})
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = req.params.id;
+  if(id <= 0){
```

Comment:
you must check also if the id is an integer: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number

check out the isInteger method